bamboo/integration_tests/performance_tests/expected_performance.csv

```diff
@@ -0,0 +1,4 @@
+model_name,           max_epoch_time, max_mean_minibatch_time, max_min_time, max_max_time, max_stdev, min_accuracy
+mnist_distributed_io, 75.00,          0.24,                    0.23,         0.67,         1.00,      97.00
+alexnet,              2000.00,        20.00,                   10.00,        20.00,        2.00,      10.00
+resnet50,             2000.00,        20.00,                   10.00,        30.00,        2.00,      30.00
```

Comment:
We should update these times with actual expected times for a model. We should also determine how many epochs each model should run, as that will affect expected times.